src/core/reference/include/openvino/reference/convert.hpp

```diff
@@ -8,6 +8,7 @@
 
 #include "ngraph/type/element_type.hpp"
 #include "ngraph/type/float16.hpp"
+#include "ngraph/type/nf4.hpp"
```

Comment:
Can `openvino` include be used here instead `ngraph` for `nf4`?

Reply:
Done